tests/regression_tests/mg_temperature/build_2g.py

```diff
@@ -0,0 +1,291 @@
+    xs_micro[900]['absorption']['O'] = np.array([7.3055E-5, 2.1850E-6])
+    xs_micro[900]['absorption']['Zr'] = np.array([4.7141E-5, 2.5941E-5])
+    xs_micro[900]['absorption']['U235'] = np.array([0.0035, 0.0749])
+    xs_micro[900]['absorption']['U238'] = np.array([0.0060, 0.0071])
+    # total
+    xs_micro[900]['total']['H'] = np.array([0.2982, 0.7332])
+    xs_micro[900]['total']['O'] = np.array([0.0885, 0.1004])
+    xs_micro[900]['total']['Zr'] = np.array([0.0370, 0.0317])
+    xs_micro[900]['total']['U235'] = np.array([0.0061, 0.0789])
+    xs_micro[900]['total']['U238'] = np.array([0.0707, 0.0613])
+    # nu-scatter matrix
+    xs_micro[900]['scatter']['H'] = np.array([[[0.0913, 0.0147],
+                                               [8.9e-08, 0.4020]]])
+    xs_micro[900]['scatter']['O'] = np.array([[[0.0812, 4.0413E-4],
+                                               [6.8186E-8, 0.0965]]])
+    xs_micro[900]['scatter']['Zr'] = np.array([[[0.0311, 3.6735E-5],
+                                                [1.3439E-8, 0.0314]]])
+    xs_micro[900]['scatter']['U235'] = np.array([[[0.0022, 2.9034E-6],
+                                                  [1.3117E-8, 0.0039]]])
+    xs_micro[900]['scatter']['U238'] = np.array([[[0.0560, 3.7619E-5],
+                                                  [1.4553E-8, 0.0538]]])
+    # nu-fission
+    xs_micro[900]['nu-fission']['U235'] = np.array([0.0059, 0.1545])
+    xs_micro[900]['nu-fission']['U238'] = np.array([0.0019, 1.1017E-7])
+    # total
+    xs_micro[900]['total']['H'] = xs_micro[900]['absorption']['H'] + \
+                                  np.sum(xs_micro[900]['scatter']['H'][0], 1)
+    xs_micro[900]['total']['O'] = xs_micro[900]['absorption']['O'] + \
+                                  np.sum(xs_micro[900]['scatter']['O'][0], 1)
+
+    xs_micro[900]['total']['Zr'] = xs_micro[900]['absorption']['Zr'] + \
+                                   np.sum(xs_micro[900]['scatter']['Zr'][0], 1)
+
+    xs_micro[900]['total']['U235'] = xs_micro[900]['absorption']['U235'] + \
+                                     np.sum(xs_micro[900]['scatter']['U235'][0], 1)
+
+    xs_micro[900]['total']['U238'] = xs_micro[900]['absorption']['U238'] + \
+                                     np.sum(xs_micro[900]['scatter']['U238'][0], 1)
+
+    # roll axis for scatter matrix
+    for t in xs_micro:
+        for n in xs_micro[t]['scatter']:
+            xs_micro[t]['scatter'][n] = np.rollaxis(xs_micro[t]['scatter'][n],
+                                                    0, 3)
+    return xs_micro
+
+
+def create_macro_dict(xs_micro):
+    """Create a dictionary with two group cross-section"""
+    xs_macro = {}
+    for t, d1 in xs_micro.items():
+        xs_macro[t] = {}
+        for r, d2 in d1.items():
+            temp = []
+            xs_macro[t][r] = {}
+            for n, v in d2.items():
+                temp.append(d2[n])
+            xs_macro[t][r]['macro'] = sum(temp)

```

Comment:
Perhaps a comment should be added explaining why the 'macro' key needed if this is already the xs_macro variable. I *think* its so you can reuse build_openmc_xs_lib for micro and macro, but clarity would be good.